packages/block-editor/src/components/block-compare/index.js

```diff
@@ -11,16 +11,21 @@ import { diffChars } from 'diff/lib/diff/character';
  * WordPress dependencies
  */
 import { __ } from '@wordpress/i18n';
-import { Component } from '@wordpress/element';
 import { getSaveContent, getSaveElement } from '@wordpress/blocks';
 
 /**
  * Internal dependencies
  */
 import BlockView from './block-view';
 
-class BlockCompare extends Component {
-	getDifference( originalContent, newContent ) {
+function BlockCompare( {
+	block,
+	onKeep,
+	onConvert,
+	convertor,
+	convertButtonText,
+} ) {
+	const getDifference = ( originalContent, newContent ) => {
```

Comment:
Ideally these are named functions so it's easier to debug

Reply:
In other words, this should be a standard function declaration, e.g. `function getDifference(...`.

Reply:
Like this ? https://github.com/WordPress/gutenberg/pull/22909/commits/26490e9903ce830315df84a15337fbb24ec15cd1

Reply:
Yep! :+1: